cmd/controller-manager/app/controller-manager.go

```diff
@@ -269,11 +269,13 @@ func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	setDuration(&duration.AvailableDelay, util.DefaultClusterAvailableDelay)
 	setDuration(&duration.UnavailableDelay, util.DefaultClusterUnavailableDelay)
 
-	election := &spec.LeaderElect
-	setString(&election.ResourceLock, util.DefaultLeaderElectionResourceLock)
-	setDuration(&election.RetryPeriod, util.DefaultLeaderElectionRetryPeriod)
-	setDuration(&election.RenewDeadline, util.DefaultLeaderElectionRenewDeadline)
-	setDuration(&election.LeaseDuration, util.DefaultLeaderElectionLeaseDuration)
+	election := spec.LeaderElect
```

Comment:
(No action required) I'm looking forward to this defaulting becoming the function of the admission webhook.